tests/unit/src/test/scala/tests/markdown/ScastieModifierSuite.scala

```diff
@@ -0,0 +1,95 @@
+package tests.markdown
+
+import mdoc.modifier.ScastieModifier
+import mdoc.StringModifier
+import mdoc.internal.cli.Settings
+
+class ScastieModifierSuite extends BaseMarkdownSuite {
+
+  private val debugClassSuffix = "test"
```

Comment:
This is needed because I generate a random UUID for suffixing the CSS class of each snippet so diff wouldn't work.